l10n_es_sigaus_invoice/models/sigaus_mixin.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2023 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import date
+
+from odoo import SUPERUSER_ID, fields, models
+
+
+class SigausMixin(models.AbstractModel):
+    _name = "sigaus.mixin"
+    _description = "Sigaus Mixin"
+
+    is_sigaus = fields.Boolean(compute="_compute_is_sigaus", store=True)
```

Comment:
The same consideration about the loading times. Pre-create the columns on installation.

Reply:
Esto debería añadirse como un pre_init_hook